app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float or int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance2: Distance or float or int) -> Distance:
```

Comment:
You can use `or` for type hint, you should use `|`. 
https://docs.python.org/3/library/typing.html#typing.Union
Fix everywhere
```suggestion
    def __add__(self, distance2: Distance | float | int) -> Distance:
```